scripts/linters/custom_eslint_checks/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "eslint-plugin-oppia",
```

Comment:
Why we have another package.json here? Can't we use the one in the root dir?

Reply:
We need it here, because this whole directory will be copied to node_modules when we run the yarn install at the beginning, without it the checks won't work.

Reply:
Why maintain two package.json file? Can you please find a way to move the required dependencies in the main package file?